p2p/discovery/message.go

```diff
@@ -58,72 +58,70 @@ func (r *rpcNode) ToNode() *Node {
 	return NewNode(r.SelfID, r.IP, r.UDPPort)
 }
 
-func byteToMsgType(byte byte) MsgType {
-	return MsgType(uint8(byte))
+func byteToMsgType(byte byte) msgType {
+	return msgType(uint8(byte))
 }
 
-func msgTypeToByte(t MsgType) byte {
+func msgTypeToByte(t msgType) byte {
 	return byte(t)
 }
 
-func generateBuff(code MsgType, encoding []byte) []byte {
+func generateBuff(code msgType, encoding []byte) []byte {
 	buff := []byte{msgTypeToByte(code)}
 
 	return append(buff, encoding...)
 }
 
-// handle send ping msg and add pending
+// handle send pong msg and add pending
 func (m *ping) handle(t *udp, from *net.UDPAddr) {
-	log.Debug("received ping from: %s", common.BytesToHex(m.SelfID.Bytes()))
+	//log.Debug("received ping from: %s", hexutil.BytesToHex(m.SelfID.Bytes()))
 
 	// response with pong
-	if m.Version != VERSION {
+	if m.Version != discoveryProtocolVersion {
 		return
 	}
 
 	resp := &pong{
 		SelfID: t.self.ID,
 	}
 
-	t.sendMsg(PongMsg, resp, from)
+	t.sendMsg(pongMsgType, resp, NewNodeWithAddr(m.SelfID, from))
 }
 
-func (m *ping) send(t *udp, to *net.UDPAddr) {
-	log.Debug("send ping msg to: %s", common.BytesToHex(m.to.Bytes()))
+func (m *ping) send(t *udp) {
+	//log.Debug("send ping msg to: %s", hexutil.BytesToHex(m.to.ID.Bytes()))
 
 	p := &pending{
 		from: m.to,
-		code: PongMsg,
+		code: pongMsgType,
 
 		callback: func(resp interface{}, addr *net.UDPAddr) (done bool) {
 			r := resp.(*pong)
 			n := NewNodeWithAddr(r.SelfID, addr)
 			t.table.updateNode(n)
 
-			log.Debug("received pong msg: %s", common.BytesToHex(r.SelfID.Bytes()))
+			//log.Debug("received pong msg: %s", hexutil.BytesToHex(r.SelfID.Bytes()))
```

Comment:
we should remove such commented statement.

Reply:
This is needed, when debug. suppress it just for debug some other part